### ISSUE TITLE
create start.jakarta.ee website

### ISSUE THREAD

**Issue by TanjaObradovic** [jakartaee/jakarta.ee#699]
Can we please add start.jakarta.ee website and link it under jakarta.ee -> resources for now? This is going to be similar to start.microprofile.io website.

**Comment by chrisguindon** [comment]
@TanjaObradovic,

Is this the source code for this new website?
https://github.com/eclipse-ee4j/starter

Do you know the requirements for hosting this new website? 

@eclipsewebmaster,
How would you handle such requests? Would we host the application or would we provision a virtual machine for the project?

**Comment by TanjaObradovic** [comment]
yes, that will be the source. here is the project page  
https://projects.eclipse.org/projects/ee4j.starter

**Comment by eclipsewebmaster** [comment]
Using Github pages may be the way to go, as there doesn't seem to be anything that looks like webcontent in that repo.

Outside of that what are the requirements, or what are they expecting?

**Comment by chrisguindon** [comment]
@eclipsewebmaster I guess we will know more soon.

I am under the impression that we want to host a java web application.

**Comment by ivargrimstad** [comment]
I think there will be some sort of application running on the serverside for the generator. It will be more or less the same requirements and setup as for https://start.microprofile.io.

**Comment by chrisguindon** [comment]
@ivargrimstad I don't believe we host start.microprofile.io. We need a bit more information before we can decide on a path forward.

**Comment by rdebusscher** [comment]
The Web application will be a Jakarta EE application which can be deployed on a Server or Runtime.

So an environment to run a Java app is required. 

start.microprofile.io. also uses a Docker environment but that is not required.

But as you can see in the repo, project is not yet very progressed yet and thus some details are not known yet.

**Comment by TanjaObradovic** [comment]
@chrisguindon was Rudy's answer helpful?

**Comment by chrisguindon** [comment]
@TanjaObradovic This looks like something that we will need to host on our OpenShift cluster.

@eclipsewebmaster any concerns?

**Comment by chrisguindon** [comment]
How is everything with this project? Are we making any progress on this?

**Comment by ivargrimstad** [comment]
We have created https://start.jakarta.ee/ as a temporary solution. We will open a new issue when we want to integrate it further into the website.
